models/articles.models.js

```diff
@@ -108,3 +115,24 @@ RETURNING comment_id, body, article_id, author, votes, created_at;
       return rows[0];
     });
 };
+
+exports.updateArticleVote = (article_id, inc_votes) => {
+  //TODO add in check if article_id works
+  const checkForID = checkArticleIdExists(article_id);
```

Comment:
No need for a checkExists here, lets allow psql to throw an error if it can't find the article first. If it doesn't we can always look at the response, and only then if that's not helpful can we consider a checkExists function but as a last resort generally 👍